src/main/java/com/google/codeu/data/TeamModel.java

```diff
@@ -0,0 +1,28 @@
+package com.google.codeu.data;
+
+public class TeamModel {
+
+    private int teamID = 0;
```

Comment:
Add a comment for each field. These models serve as our schema so its very important that it they're well documented. Yes, the names are self explanatory but there are other pieces of information that could be included as well. For example for teamId you could d:

```java
// The id of this team. Assigned by the database when the team is first created.`
private UUID teamId;
```